tests/e2e/k8s/test-pod.yaml.tpl

```diff
@@ -10,3 +11,5 @@ spec:
       command: ["sh", "-c"]
       args:
         - tail -f /dev/null
+  imagePullSecrets:
+    - name: regcred
```

Comment:
Add a new line at the end, otherwise, GitHub will show a `No newline at end of file` message.